benches/encoding_primitives.rs

```diff
@@ -0,0 +1,213 @@
+use criterion::{black_box, criterion_group, criterion_main, Criterion};
+use rand::prelude::StdRng;
+use rand::{distributions::Uniform, Rng, SeedableRng};
+use std::io;
```

Comment:
🗺️ This file is a suite of benchmarks comparing the time needed to read or write 10k random integers using `VarUInt`, `VarInt`, `FlexUInt`, or `FlexInt` encoding.